test.sh

```diff
@@ -743,6 +750,24 @@ fi
 
 # ---- Rest of the coverage runs included in CI test runs ----
 
+# ------------------------------------------------------------------------
+# Fast-path execution support. You can invoke this script specifying the
+# name of one of the functions to execute a specific set of tests. If the
+# function takes arguments, pass them on the command-line. This way, one
+# can debug script changes to ensure that test-execution still works.
+#
+# Examples:
+#      INCLUDE_SLOW_TESTS=true ./test.sh run_btree_tests
+# ------------------------------------------------------------------------
+if [ $# -ge 1 ]; then
+
+   # shellcheck disable=SC2048
+   $*
```

Comment:
This is where the support to re-run manually granular collection of tests is added. This will help expedite dev turnaround during fix stabilization and re-tests.